_episodes/08-filters.md

```diff
@@ -3,11 +3,17 @@ title: "Working with Filters"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "How can I specify the format of interpolated variables?"
```

Comment:
Could we reword this one?  From my lake of jekyll knowledge, I'm not sure I quite understand the question.

Reply:
What about

> How can I control the format of variable values such as dates when inserting them into my pages?

Reply:
I think that sounds great!